modin/experimental/core/execution/native/implementations/omnisci_on_native/test/test_dataframe.py

```diff
@@ -466,6 +470,7 @@ def applier(df, **kwargs):
         run_and_compare(
             fn=applier,
             data=data,
+            force_lazy=False,
```

Comment:
The necessity to add `force_lazy=False` to some of the existing tests means you introduce a degradation. Lazy execution is of extreme importance for OmniSci backend. It can never provide enough performance with lazy execution broken due to high overhead for data import/export.

I also don't see why the new test shouldn't have lazy execution enabled.